### PR TITLE
Add picocad to chromebrew

### PR DESCRIPTION
**Pull request by Zinc-OS**
Works properly:
- [x] x86_64

---

### CODE REVIEW

File: packages/picocad.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Picocad < Package                 # The first character of the class name must be upper case
+  description 'picoCAD is a program to build and texture lowpoly 3D models. Where many programs for modelling and texturing are bloated and overly complicated, picoCAD aims to make it fun, easy, and accessible by focusing on the bare essentials. It is built on the PICO-8 platform and comes rich with constraints. Experiment to find your own workflow and anything is possible!'
```
Comment (uberhacker) [inline packages/picocad.rb:4]: Can we shorten the description to the first sentence?

File: packages/picocad.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  homepage 'https://johanpeitz.itch.io/picocad'
+  version '1.01'
+  compatibility 'all'
+  depends_on 'sommelier'            # packages required by this package
```
Comment (uberhacker) [inline packages/picocad.rb:8]: Please put this line below `depends_on 'libx11'` so users don't miss the postinstall message.

File: packages/picocad.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Picocad < Package                 # The first character of the class name must be upper case
+  description 'picoCAD is a program to build and texture lowpoly 3D models. Where many programs for modelling and texturing are bloated and overly complicated, picoCAD aims to make it fun, easy, and accessible by focusing on the bare essentials. It is built on the PICO-8 platform and comes rich with constraints. Experiment to find your own workflow and anything is possible!'
+  homepage 'https://johanpeitz.itch.io/picocad'
+  version '1.01'
```
Comment (uberhacker) [inline packages/picocad.rb:6]: Missing `license 'proprietary'`?

File: packages/picocad.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  compatibility 'all'
+  depends_on 'sommelier'            # packages required by this package
+  depends_on 'libx11'            # packages required by this package
+  source_url 'https://cdn.discordapp.com/attachments/768193037708558427/823684213093171220/picocad-101.zip'
+  source_sha256 '13da7c39d4c78d4878bc00fbe59ac3865852c60b4ad72b97cfd33f5619b6feb6'   # Use the command "sha256sum"
+
+  def self.install                 # the steps required to install the package
+    system "install picocad-101.bin/linux/picocad -Dt /usr/local/bin -m 755"
+    system "install picocad-101.bin/linux/data.pod -Dt /usr/local/bin -m 755"
```
Comment (uberhacker) [inline packages/picocad.rb:15]: Please change to:
```
FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
FileUtils.install 'picocad-101.bin/linux/picocad', "#{CREW_DEST_PREFIX}/bin/picocad", mode:0755
FileUtils.install 'picocad-101.bin/linux/data.pod', "#{CREW_DEST_PREFIX}/bin/data.pod", mode:0644
```
This directory is necessary to make sure the files are tracked during install.

**Comment by uberhacker** [comment]
Thank you for the new package @Zinc-OS.  Please see the comments.

File: packages/picocad.rb (hunk 3 above)
Comment (Zinc-OS) [reply packages/picocad.rb:6]: of course

**Review by supechicken — APPROVED** [review]

File: packages/picocad.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Picocad < Package                 # The first character of the class name must be upper case
+  description 'picoCAD is a program to build and texture lowpoly 3D models.'
+  homepage 'https://johanpeitz.itch.io/picocad'
+  version '1.0.1'
+  compatibility 'all'
+  license 'Proprietary'
```
Comment (uberhacker) [inline packages/picocad.rb:8]: @saltedcoffii: What are your thoughts about this license?

File: packages/picocad.rb [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  depends_on 'libx11'            # packages required by this package
+  depends_on 'sommelier'            # packages required by this package
+  depends_on 'libsdl2'
```
Comment (uberhacker) [inline packages/picocad.rb:11]: Please make `depends_on 'sommelier'` the last line.

File: packages/picocad.rb (hunk 5 above)
Comment (saltedcoffii) [reply packages/picocad.rb:8]: It needs to specify the TOS or EULA specifically. Just stating that a license is proprietary isn't enough.

**Comment by saltedcoffii** [comment]
Also don't forget to remove the template comments from the final package

**Comment by satmandu** [comment]
Is this still in limbo? Would you please ask the project author what the license for this software is? Also it appears that he would like people to download this through a[ link on his site which allows people to donate to him using "name your own price"](https://johanpeitz.itch.io/picocad/purchase), so I'm also not sure he would be happy with an independent distribution of the software.

**Pull request closed without merging**